src/envs/behavior.py

```diff
@@ -454,26 +465,19 @@ def _initiable(state: State, memory: Dict, objects: Sequence[Object],
                    params: Array) -> bool:
         igo = [object_to_ig_object(o) for o in objects]
         assert len(igo) == 1
-        if memory.get("policy_controller") is None:
+        if memory.get("rrt_plan") is None:
```

Comment:
The logic is written in a bit of a weird way here with the hanging `return True` at the end. Instead can we do:
```
if memory.get("rrt_plan") is not None:
    # TODO: add a comment here explaining what exactly it means if we are here
    return True
# We want to reset the state of the environment to
etc etc etc
```